### PR TITLE
docs: support Valine Comment System

### PR DESCRIPTION
**Pull request by HerbertHe**

### CODE REVIEW

File: zh/customization.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ content/
 
 ## 评论
 
-Eureka支持使用Disqus，Utterances和Commento作为评论系统。如需配置评论，只需对`config/_default/params.yaml`中的`comment`进行相关配置即可。
+Eureka支持使用Disqus，Utterances，Valine和Commento作为评论系统。如需配置评论，只需对`config/_default/params.yaml`中的`comment`进行相关配置即可。
```
Comment (wangchucheng) [inline zh/customization.md:90]: Please add the detail explanation of valine. You can refer to `comment.utterances` or `diagram.mermaid` for example.

File: en/customization.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ You can set the value of the `math` attribute in the Front Matter of the file in
 
 ## Comment
 
-Eureka supports using Disqus, Utterances and Commento as comment system。If you need to configure comment, you just need to configure the `comment` in `config/_default/params.yaml`
+Eureka supports using Disqus, Utterances, Valine and Commento as comment system。If you need to configure comment, you just need to configure the `comment` in `config/_default/params.yaml`
```
Comment (wangchucheng) [inline en/customization.md:90]: Please add the detail explanation of valine. You can refer to `comment.utterances` or `diagram.mermaid` for example.

**Review by wangchucheng — CHANGES_REQUESTED** [review]
Thanks! Please add some additional info about valine as below.

**Review by HerbertHe — COMMENTED** [review]
reviewed

**Review by HerbertHe — COMMENTED** [review]
viewed

**Comment by wangchucheng** [comment]
Thanks for this pr. But this pr is only about adding valine support, so I can not merge other changes. And the docs have been added in e5ac227. So I will close this PR.

If you think the document needs to be improved, please feel free to propose a new related pr. Thanks!

**Pull request closed without merging**